### PR TITLE
Feat - Added post functionalities

### PR DESCRIPTION
**Pull request by SRUJANA-PENUGONDA13**
## Added Post Functionalities
1. Edit Post
2. Delete Post
3. Save and Un Save Post
4. Explore posts
5. Trending and Latest Posts Filter

https://user-images.githubusercontent.com/72279128/183683008-ca17bc11-7aee-4af0-9e6d-57ab48bb7eb0.mp4

### CODE REVIEW

**Review by SanketDhabarde — APPROVED** [review]
- Code looks good
- The UI is nice
- Good to merge

**Review by NeerajVodala — APPROVED** [review]

File: src/frontend/components/postsContent/PostsContent.css [hunk 1]
```diff
@@ -7,6 +7,30 @@
...
+}
+
+.tablist {
+  color: var(--text--primary--color);
+  width: 100%;
+  gap: 5rem;
+  padding: 0rem 13rem;
+  align-items: center;
+  justify-content: center;
+  border: none !important;
```
Comment (ashishkatiyar2404) [inline src/frontend/components/postsContent/PostsContent.css:22]: you can remove it !important and try  others method.
you can increase specificity.

File: src/frontend/components/postsContent/PostsContent.css [hunk 2]
```diff
@@ -7,6 +7,30 @@
...
+}
+
+.tab-btn:hover {
+  border: 1px solid var(--text--primary--color) !important;
+  color: var(--text--primary--color);
+}
+
+.posts-control-btn {
+  background-color: var(--primary--color) !important;
```
Comment (ashishkatiyar2404) [inline src/frontend/components/postsContent/PostsContent.css:31]: here also

**Review by ashishkatiyar2404 — APPROVED** [review]
well structured code looks good to merge

**Pull request merged**